src/main/java/htsjdk/samtools/util/AbstractRecordAndOffset.java

```diff
@@ -124,41 +113,20 @@ public String getReadName() {
     public byte[] getBaseQualities() {
         return record.getBaseQualities();
     }
-
+    
+    /**
+     * @return the base quality according to <code>offset</code>.
+     */
+    public byte getBaseQuality() {
+        return record.getBaseQualities()[offset];
```

Comment:
so this is the base quality of the base at the "start" of the AbstractRecordAndOffset, yes?